day_7/projects/10_speckled_frogs.rb

```diff
@@ -0,0 +1,51 @@
+frogs = [10, 9, 8, 7, 6, 5, 4, 3, 2, 1]
```

Comment:
@jakejakearell - While testing out this code, the rhyme correctly prints out counting down from 10, however I noticed a couple issues, specifically around wording - for example, the first iteration prints out:

> 10 speckled frogs sat on a log eating some most delicious bugs.  One jumped in the pool where its nice and cool, then there was 9 speckled frogs.

The issue here is that it should say "there **were** 9 speckled frogs".  The same issue happens again throughout.  When we get to the second to last iteration, it prints out `"then there was 1 speckled frogs."` Since there's only 1 left, this should actually say "frog" singular.

How could you update this exercise to correctly print out each iteration of the rhyme?